src/templates/ar/2019/base.html

```diff
@@ -65,6 +65,7 @@
         {{ language_switcher() }}
       </nav>
       <p class="copyright"><span>© 2019 Web Almanac. Licensed under <a href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/master/LICENSE">Apache 2.0</a>.</span></p>
```

Comment:
WDYT of something like this instead?

```suggestion
      <p class="copyright"><span>© 2019 Web Almanac. Licensed under <a href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/master/LICENSE">Apache 2.0</a>.</span><br>
      Read our <a>Accessibility Statement</a>.</p>
```

Reply:
I like it!

Fixed in https://github.com/HTTPArchive/almanac.httparchive.org/pull/665/commits/1638e53c39d5a178076de0e81a8e981a6f29bd9e